pkg/config/config.go

```diff
@@ -118,6 +123,15 @@ func (c *Config) ShouldIgnorePackageVersion(name, version, ecosystem string) (bo
 	})
 }
 
+// ShouldIgnorePackageVulnerabilities determines if the given package should have its vulnerabilities ignored based on override entries in the config
+func (c *Config) ShouldIgnorePackageVulnerabilities(pkg models.PackageVulns) bool {
+	overrides, _ := c.filterPackageVersionEntries(pkg, func(e PackageOverrideEntry) bool {
+		return e.Vulnerability.Ignore
+	})
+
+	return overrides
+}
```

Comment:
note: the names here don't really make much sense but I think that's a bigger issue that should be tackled at the package level

I also didn't keep the interface matching the other functions because I couldn't think of _any_ unsurprising way of leveraging "overrides for vulnerabilities" like with licenses, and I feel like having this be as specific as possible should make it easier to do non-breaking changes in future

i.e. my hope is that if in future we do think up more uses for this struct, we'll be able to introduce `ShouldOverridePackageVulnerabilities` and co, and just deprecate this method

Given this is a public method, I am happy to switch this to returning the override entry to be on the safe side anyway given there's no major downside to doing that straight away even if we never use it